rel-eng/tito.props

```diff
@@ -341,8 +341,6 @@ whitelist = rubygem-angular-rails-templates
   rubygem-diffy
   rubygem-down
   rubygem-ffi
-  rubygem-foreman-tasks
```

Comment:
can you please revert the sorting changes? we really need to make this script locale independant…